src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball randomBall = new Ball(Color.BLUE, 0);
+
+        for (int i = 0; i < 3; i++) {
+            randomBall = lottery.getRandomBall();
+            System.out.println(randomBall.toString());
```

Comment:
You can use `System.out.println(randomBall)` directly. Java will automatically call the `toString` method for you.

Reply:
Thank you for tip. Done it.